### DISCUSSION TITLE
Graph showing as a flat line at the end of a roast

### DISCUSSION THREAD

**Discussion by NathProsser** [artisan-roaster-scope/artisan#1762] [General]
Hey, folks.

Hoping someone can assist with some knowledge here...

During a roast, my graph looks normal.  And when I click "reset" or any other button to start another roast, it shows the last complete roast graph prior to saving.  That graph then appears as a totally flat line.  I just updated to most recent version and still happening.

Roasting on coffee-tech roasters.

Any support is really useful!

Thanks, 

Nath

**Comment by roasterdave** [comment]
This is quite odd.  From the description it sounds like this happened also with a previous version of Artisan before you upgraded to v3.1.0, is that correct?

To help further requires some additional information.  Please provide an saved profile (alog file) where this happened and your settings file (Help>> Save Settings).  Rename both files to have a ".txt" suffix (example: artisan-settings.aset.txt and myprofile.alog.txt).  Then drag and drop them onto a reply box here on GitHub (do not attach them to an email reply).

A screenshot of the flat line graph you described would also help.  Again, you may drag and drop an image here.
